src/Components/Components/src/Microsoft.AspNetCore.Components.WarningSuppressions.xml

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="Microsoft.AspNetCore.Components, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2026</argument>
```

Comment:
It's not obvious from reading this file what warnings are being suppressed. What are IL2026 and the others? Could you add comments stating what is the purpose of the suppressions?

Reply:
The files were generated by the linker so it'd be a bit of an upkeep to document it. A lot of these are complaining about the various reflection patterns with components and I'm hoping we get a [linker feature](https://github.com/mono/linker/issues/1806) that removes this need.